backend/classification/predict.py

```diff
@@ -16,8 +20,18 @@ def predict(raw_eeg, info):
             the subject started the recording and went to bed
         - out_of_bed_seconds: timespan, in seconds, from which
             the subject started the recording and got out of bed
+    Returns: array of predicted sleep stages
     """
+    classifier, postprocessing_state = model['classifier'], model['postprocessing']
+
     validate(raw_eeg, info)
-    X_openbci = get_features(raw_eeg, info)
+    features = get_features(raw_eeg, info)
+    input_name = classifier.get_inputs()[0].name
+
+    predictions = classifier.run(None, {input_name: features})[0]
+    predictions = postprocess(predictions, postprocessing_state)
+
+    print(features[0], features.shape)
+    print(predictions)
```

Comment:
is this intentional?

Reply:
yes, as we still return the mock (will change to real data when working on calculating the metrics in another PR) and I wanted a way to see the results meanwhile.